modules/rosa-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,61 @@
+:_module-type: PROCEDURE
+//Specify the _module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// rosa-nodes/rosa-autoscale-nodes.adoc
+
+[id="rosa-autoscaling_{context}"]
+= Autoscaling nodes in a cluster
+
+[role="_abstract"]
+Configure autoscaling to increase the number of worker nodes that are available in a cluster. Autoscaling for worker nodes is configured in the machine pool definition.
```

Comment:
Maybe consider a more active voice for the second sentence?

"You can configure autoscaling for worker nodes in the machine pool definition."

Reply:
I considered that originally. but that makes is seem like it's possible to configure node autoscaling another way. I want to make it clear that this is the only way to do it. Will leave as-is for now.